streams/writable-streams/aborting.js

```diff
@@ -157,16 +163,15 @@ promise_test(t => {
 
   const writer = ws.getWriter();
 
-  writer.write().catch(() => {});
+  writer.write(1).catch(() => {});
```

Comment:
In ECMAScript, calling write() and doing write(undefined) are the same, so it's not possible for us to spec something that handles write() differently. This is already in the spec.